run/ssh2john.py

```diff
@@ -45,6 +45,27 @@
     'AES-256-CTR': {'cipher': AES_256, 'keysize': 32, 'blocksize': 16, 'mode': "AES.MODE_CTR"},
 }
 
+def get_all_tags_ktypes(lines):
+    tags = []
+    ktypes = []
+    for line in lines:
+        if line.find("BEGIN RSA PRIVATE") > 0:
```

Comment:
I think `if "BEGIN RSA PRIVATE" in line:` is more idiomatic Python. But I don't know how important comments on style are.

Reply:
Idiomatic python is the way to go, thanks :)